scipy/interpolate/_bspl.pyx

```diff
@@ -457,4 +457,4 @@ def _make_design_matrix(const double[::1] x,
                                                             ,min(ind + 1, nt - k - 1)
                                                             ,dtype=int)
 
-    return csr_matrix((data, (row_ind, col_ind)), (n, nt - k - 1))       
+    return csr_matrix((data.base, (row_ind, col_ind)), (n, nt - k - 1))       
```

Comment:
This is a bit obscure, would it work to use `np.asarray(data) `

Reply:
Yes, it would! Btw, as far as the problem was not in `csr_matrix` constructor I've decided to keep `np.asarray(data)` to pass to `csr_matrix` a Python object rather that a `memoryview`.